time_utils_unittest.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file time_utils_unittest.cpp
```

Comment:
I have written only one unit test in my 1.5 years as a firmware engineer...

startups...

Reply:
I'm in the unit test soup every day